fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -30,14 +30,28 @@ import { makeRollCallSelector } from '../reducer';
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    alignItems: 'center',
     justifyContent: 'space-between',
     marginVertical: Spacing.contentSpacing,
   } as ViewStyle,
   qrCode: {
```

Comment:
I am not totally sure to understand the question but yes `qrCode` is still used, the reason why there's multiple `alignItems: 'center'` is because we want all items except the counter to be centered, so I add to make the centered property in less higher level in the code